pkg/kubectl/cmd/portforward/portforward.go

```diff
@@ -59,7 +59,7 @@ type PortForwardOptions struct {
 
 var (
 	portforwardLong = templates.LongDesc(i18n.T(`
-                Forward one or more local ports to a pod.
+                Forward one or more local ports to a pod. This command requires node has 'socat' installed.
```

Comment:
please change the message to:
`This command requires the node to have 'socat' installed.`